release/sil/sil.km.ggc/source/sil.km.ggc.model.ts

```diff
@@ -10,59 +10,71 @@
 const source: LexicalModelSource = {
   format: 'trie-1.0',
   sources: ['wordlist.tsv'],
-        wordBreaker: function (str) {
-  const tokens = str.split(/\s|\u200b/);
+  wordBreaker: function (str) {
+    const whitespaceRegex = /\s|\u200b|\n|\r/;
+    const tokens = str.split(whitespaceRegex);
 
-  for(let i=0; i < tokens.length; i++) {
-    const token = tokens[i];
-    if(token.length == 1) {
-      continue;
-    }
+    for(let i=0; i < tokens.length; i++) {
+      const token = tokens[i];
+      if(token.length == 0) {
+        tokens.splice(i, 1);
+        i--;
+        continue;
+      } else if(token.length == 1 && whitespaceRegex.test(token)) {
+        tokens.splice(i, 1);
+        i--;
+        continue;
+      }
```

Comment:
Prevents empty tokens from appearing in the middle of context, especially should excessive whitespaces exist between words.  Something like `the        quick   brown fox` should appear to skip all spaces between tokens at once, not in multiple steps.

Reply:
Does the custom wordbreaker for sil.km.cnd.model.ts need a similar fix?

https://github.com/keymanapp/lexical-models/blob/a90fec5d323bc30f6fe1783ec604aab95f3d93a2/release/sil/sil.km.cnd/source/sil.km.cnd.model.ts#L13-L25